test/addons-napi/test_string/test.js

```diff
@@ -46,7 +46,8 @@ const str4 = '¡¢£¤¥¦§¨©ª«¬­®¯°±²³´µ¶·¸¹º»¼½¾¿';
 assert.strictEqual(test_string.TestLatin1(str4), str4);
 assert.strictEqual(test_string.TestUtf8(str4), str4);
 assert.strictEqual(test_string.TestUtf16(str4), str4);
-assert.strictEqual(test_string.TestLatin1Insufficient(str4), str4.slice(0, 3));
+// TODO: https://github.com/nodejs/abi-stable-node/issues/235
+// assert.strictEqual(test_string.TestLatin1Insufficient(str4), str4.slice(0, 3));
```

Comment:
/home/kfarnung/GitHub/node-chakracore/test/addons-napi/test_string/test.js
  50:1  error  Line 50 exceeds the maximum line length of 80  max-len
  61:1  error  Line 61 exceeds the maximum line length of 80  max-len